frontend/src/views/BookingView.vue

```diff
@@ -183,9 +183,10 @@ const bookEvent = async (attendeeData) => {
     attendee: attendeeData,
   };
 
+  const url = window.location.href.split('#')[0];
```

Comment:
Are we guaranteed to have a '#' in the href here?

Reply:
submitted too early. What happens if we get an error here?

Reply:
split will still return an array with the url. if theres a hash it will be length==2 instead of 1.

Reply:
oh whoops my bad, thanks!